### PR TITLE
Partial update of repository docs.

### PR DESCRIPTION
**Pull request by jeromekelleher**
This is a work in progress updating the repository administration docs. Here's what it looks like on readthedocs: http://ga4gh-server-fork.readthedocs.io/en/1205-admin-cli-docs/configuration.html#data-repository

Please review this and give some feedback.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Any opinions? It would be very helpful to get some feedback on what's been done so far. The RTD link about should make it easy to see what it all looks like.

File: docs/configuration.rst [hunk 1]
```diff
@@ -99,183 +129,244 @@ Creates a dataset in the given repository with a given name.
...
 
-++++++++++++++++++++
-remove-referenceset
-++++++++++++++++++++
++++++++++++++++
+add-variantset
++++++++++++++++
 
-Removes a given reference set from a given data repository.
+Adds a variant set to a names dataset in a repository. Variant sets are
```
Comment (david4096) [inline docs/configuration.rst]: *named

File: docs/configuration.rst [hunk 2]
```diff
@@ -99,183 +129,244 @@ Creates a dataset in the given repository with a given name.
...
+currently derived from one or more non-overlapping VCF/BCF files which
+may be either stored locally or come from a remote URL. Multiple VCF
+files can be specified either directly on the command line or by
+providing a single directory argument that contains indexed VCF files.
+If remote URLs are used then a index files in the local file system must be
```
Comment (david4096) [inline docs/configuration.rst]: strike *a

**Comment by david4096** [comment]
@jeromekelleher Looks good to me. Slight refactor and a documentation PR +1

**Comment by kozbo** [comment]
I have a structural suggestion, to put the "remove" commands nest  to their respective add commands, and to put the "list" and "verify" commands at the end.

**Comment by jeromekelleher** [comment]
Thanks @david4096, good catches.

@kozbo I don't know, this was the way things were laid out and it just seemed a bit easier to see where things were when they were grouped by method rather than object. It is a bit easier to maintain things when all the 'remove' commands are together, as they're almost identical and you can see if they are inconsistent when they are next to each other. I don't have a strong opinion on it though, happy to change it back.

**Comment by kozbo** [comment]
@jeromekelleher OK, I see the logic now, thanks. +1

**Comment by jeromekelleher** [comment]
Updated to add information on adding readgroup sets, and clarify the remove-x commands. This should complete the first pass, and be ready merge.

**Pull request merged**